core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -126,7 +126,7 @@ THE SOFTWARE.
                 </st:include>
                 <td class="pane" align="center" valign="middle">
                   <j:if test="${h.hasPermission(exeparent,exeparent.READ) and e.hasStopPermission()}">
-                    <l:stopButton href="${rootURL}/${c.url}${url}/stopBuild?runExtId=${h.urlEncode(exe.externalizableId)}" confirm="${%confirm(exe.fullDisplayName)}" alt="${%terminate this build}"  />
+                    <l:stopButton href="${rootURL}/${c.url}${url}/stopBuild?runExtId=${h.urlEncode(exe.externalizableId)}" confirm="${%confirm(exe.fullDisplayName)}" alt="${%Cancel}"  />
```

Comment:
This will break the i18n, it would be better to use Cancel in the properties as {en} translation. Or rename all translations.